xml/System.Windows.Annotations/IAnchorInfo.xml

```diff
@@ -15,7 +15,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.Windows.Annotations.IAnchorInfo> is introduced in the [!INCLUDE[TLA#tla_netframewk](~/includes/tlasharptla-netframewk-md.md)] version 3.5.  For more information, see [Versions and Dependencies](~/docs/framework/migration-guide/versions-and-dependencies.md).  
+ <xref:System.Windows.Annotations.IAnchorInfo> is available starting with .NET Framework version 3.5.  For more information, see [Versions and Dependencies](~/docs/framework/migration-guide/versions-and-dependencies.md).  
```

Comment:
sentence here looks fine but same question applies here and to all other changed remarks